### DISCUSSION TITLE
cases_small is exceeding the GAMS academic version

### DISCUSSION THREAD

**Discussion by inunezfe** [NREL/ReEDS-2.0#140] [Q&A: Execution] [answered]
Hi ReEDS team

I have been trying to run cases_small.csv to see if the model works on my computing hardware, but the gamslog.txt file that is created after the execution shows that i have exceeded the 5000 rows and columns, could you help me out?

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/111766958/218518281-f52960e7-5608-411d-b363-f775d8dccb50.png">

Thanks

**Comment by wesleyjcole** [comment, marked as answer]
I just pushed a change into Main that should allow you to run with cases_small.csv and stay under the 5,000 limit.  Let me know if you have any issues with getting it to run.
